### PR TITLE
Updates of documentation improvements

### PR DESCRIPTION
**Pull request by alexandesigner**
## Fix Readme duplication

Now we're 3 Readme models:

- Specific for the Basic Package Blaze in Meteor; [Package Readme](https://github.com/alexandesigner/blaze/blob/master/packages/blaze/README.md)
- Main Readme is what will be seen by accessing the repository; [Main Readme](https://github.com/alexandesigner/blaze/blob/master/README.md)
- Extensive Readme for the introduction in the documentation (Site); [Extensive Readme](https://github.com/meteor/blaze/blob/master/site/source/index.md)

_Everyone has a certain similarity, but it's not a duplicate version._

## Import themes

Instead of using a custom theme, now we are importing the main theme of the Meteor using ``.gitsubmodule``

Improved version of the theme ... We are no longer using the customized version..

## Organization

Organizing your files within ``/source``.. Created paths ``/api`` and ``/guide``.

### CODE REVIEW

**Comment by mitar** [comment]
"This branch has conflicts that must be resolved". Please resolve conflicts.

File: .gitmodules [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [submodule "tests/test-runner"]
 	path = tests/test-runner
 	url = https://github.com/tozd/meteor-test-runner.git
-[submodule "sites/themes/meteor"]
-	path = sites/themes/meteor
-	url = https://github.com/meteor/hexo-theme-meteor.git
+[submodule "site/themes/meteor"]
+	path = site/themes/meteor
+	url = https://github.com/meteor/hexo-theme-meteor
```
Comment (mitar) [inline .gitmodules]: Use URL with `.git` at the end.

Comment (alexandesigner) [inline .gitmodules]: is it really necessary? Just ran the command "git submodule add.."

Comment (mitar) [inline .gitmodules]: Oh, this is just style. GitHub will handle it correctly with a redirect.

Comment (alexandesigner) [inline .gitmodules]: All right 😄

**Comment by mitar** [comment]
Thanks!

**Pull request merged**